sklearn/metrics/scorer.py

```diff
@@ -226,11 +248,14 @@ def get_scorer(scoring):
     """
     if isinstance(scoring, str):
         try:
-            scorer = SCORERS[scoring]
+            if scoring == 'brier_score_loss':
+                scorer = brier_score_loss_scorer
+            else:
+                scorer = SCORERS[scoring]
         except KeyError:
             raise ValueError('%r is not a valid scoring value. '
                              'Use sorted(sklearn.metrics.SCORERS.keys()) '
-                             'to get valid options.' % (scoring))
+                             'to get valid options.' % scoring)
```

Comment:
revert, please.

Reply:
Why revert? Those parentheses have no effect. 

Might be better just to use f-strings these days

Reply:
I know it's better. I just want to avoid some irrelevant changes. Ignore this review please.